src/components/WelcomeBox.js

```diff
@@ -53,18 +53,19 @@ class WelcomeBox extends Component {
     const searchFunctionLabel = L10N.getStr("welcome.searchFunction").substring(
       2
     );
+    const { setActiveSearch } = this.props;
 
     return (
       <div className="welcomebox">
         <div className="alignlabel">
           <div className="shortcutKeys">
-            <p>{searchSourcesShortcut}</p>
-            <p>{searchProjectShortcut}</p>
+            <p onClick={setActiveSearch.bind(null, "source")}>{searchSourcesShortcut}</p>
+            <p onClick={setActiveSearch.bind(null, "project")}>{searchProjectShortcut}</p>
             <p>{searchFunctionsShortcut}</p>
```

Comment:
I really think we should remove this shortcut.
The other shortcuts are "actionable" at this point but search for functions can not be called from WelcomeBox because it  is only applicable on opened source.
Any thoughts @jasonLaster  @codehag

Reply:
I agree! The symbol modal hint doesn't make sense here.